test/typescript/axios.ts

```diff
@@ -130,7 +138,7 @@ axios.get<User>('/user', { params: { id: 12345 } })
 axios.head<User>('/user')
 	.then(handleUserResponse)
     .catch(handleError);
-    
+
```

Comment:
If this is not related to your change, maybe don't include this, to keep the PR simpler.

Reply:
thanks Lucas. I will probably move everything to a new branch so it would only have commit related to the type changes